### PR TITLE
Docs: add Integrations section

### PR DESCRIPTION
**Pull request by LaverdeS**
Created `integrations.rst` with intro and 8 integrations, and added that to the index. I only included code for LangChain integration (not visible in github preview though). The other integrations have a simple descriptions with capabilities, some parameters usage, and links to the "integration company"/github and to the code snippet in our staging bricks docu. Here is the list:
- Arguila
- Datasaur
- Hugging Face
- Labelbox
- Label Studio
- LangChain
- Pandas
- Prodigy

### CODE REVIEW

File: docs/source/integrations.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+``Integration with Pandas``
+--------------
+You can convert a list of document Element objects to a Pandas dataframe with a ``text`` and ``type`` columns with 
+the text from the element and its type such as ``NarrativeText`` or ``Title``. Just follow the link for our staging brick 
+`convert_to_dataframe <https://unstructured-io.github.io/unstructured/bricks.html#convert-to-dataframe>`_ to see how to use it.
+
+
+``Integration with Prodigy``
+--------------
+You can format your outputs in a JSON or CSV format for use with `Prodigy <https://prodi.gy/docs/api-loaders>`_. After running ``stage_for_prodigy`` |
```
Comment (qued) [inline docs/source/integrations.rst:74]:
```suggestion
You can format your JSON or CSV outputs for use with `Prodigy <https://prodi.gy/docs/api-loaders>`_. After running ``stage_for_prodigy`` | 
```

File: docs/source/integrations.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+--------------
+You can convert a list of document Element objects to a Pandas dataframe with a ``text`` and ``type`` columns with 
+the text from the element and its type such as ``NarrativeText`` or ``Title``. Just follow the link for our staging brick 
+`convert_to_dataframe <https://unstructured-io.github.io/unstructured/bricks.html#convert-to-dataframe>`_ to see how to use it.
+
+
+``Integration with Prodigy``
+--------------
+You can format your outputs in a JSON or CSV format for use with `Prodigy <https://prodi.gy/docs/api-loaders>`_. After running ``stage_for_prodigy`` | 
+``stage_csv_for_prodigy``, you can write the results to a ``.json`` | ``.jsonl`` or a ``.csv`` file. that is ready to be used with Prodigy. Just follow the link for our staging bricks `stage_for_prodigy <https://unstructured-io.github.io/unstructured/bricks.html#stage-for-prodigy>`_ and
```
Comment (qued) [inline docs/source/integrations.rst:75]:
```suggestion
``stage_csv_for_prodigy``, you can write the results to a ``.json`` | ``.jsonl`` or a ``.csv`` file that is ready to be used with Prodigy. Just follow the links for our staging bricks `stage_for_prodigy <https://unstructured-io.github.io/unstructured/bricks.html#stage-for-prodigy>`_ and 
```

**Review by qued — APPROVED** [review]
LGTM, just small changes. Additionally, I think it'd be better to structure these by introducing the function by name near the beginning of the paragraph. I've provided a few examples of what I mean, but this is just a suggestion.

File: docs/source/integrations.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
+Integrations
+======
+Integrate your model development pipeline with your favorite machine learning frameworks and libraries, 
+and prepare your data for ingestion into downstream systems. Most of out integrations are in the form of
```
[the remark this answers is not in the document]
Comment (LaverdeS) [reply docs/source/integrations.rst:4]: There is only one that is not a staging brick and that LangChain

File: docs/source/integrations.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+some of our `staging bricks <https://unstructured-io.github.io/unstructured/bricks.html#staging>`_, 
+design to get a list of document ``unstructured`` ``Element`` objects as input and return formatted dictionaries as output.
+
+
+``Integration with Arguila``
+--------------
+You can convert a list of ``Text`` elements to an `Argilla <https://www.argilla.io/>`_ Dataset by specifying the type of 
+dataset to be generated using the ``argilla_task`` parameter. Valid values are "text_classification", "token_classification", and "text2text".
+Just follow the link for our staging brick `stage_for_arguila <https://unstructured-io.github.io/unstructured/bricks.html#stage-for-argilla>`_ 
+to see how to do it.
```
[the remark this answers is not in the document]
Comment (LaverdeS) [reply docs/source/integrations.rst:14]: Thank you, I will apply this to all the integrations then.

**Comment by LaverdeS** [comment]
Pending suggestion [here](https://github.com/Unstructured-IO/unstructured/pull/372#discussion_r1139018253).

**Pull request merged**